.github/workflows/go.yml

```diff
@@ -6,7 +6,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go: [1.13, 1.14]
+        go: [1.13, 1.14, 1.15]
```

Comment:
I do not think we should be testing on three versions.  The policy has always been current and current-1.  So if 1.15 is current, please drop 1.13.

Reply:
@jcvernaleo should be good to go here